packages/lib-classifier/src/store/TutorialStore.js

```diff
@@ -199,11 +199,11 @@ const TutorialStore = types
       }
     }
 
-    function setActiveTutorial (id, stepIndex) {
-      if (!id) {
+    function setActiveTutorial (tutorial, stepIndex) {
+      if (!tutorial) {
         self.resetActiveTutorial()
       } else {
-        self.active = id
+        self.active = tutorial
```

Comment:
This is simply re-labeling the active tutorial in the store for clarity

Reply:
`self.active` is a reference, so it should be set to the identifier of the model that it references.

Reply:
Yeah. I've changed that in #2675 too. Looking through the other stores, we're actually inconsistent as to whether we pass identifiers as references. Some models do, others don't. I'm 90% certain that MST references should always be identifiers. They are certainly serialised as IDs in snapshots.

MST isn't throwing invalid reference errors, so I don't think this is a blocking change, by the way,